.github/workflows/release_docker.yml

```diff
@@ -0,0 +1,114 @@
+# **what?**
+# This workflow will generate a series of docker images for dbt and push them to the github container registry
+
+# **why?**
+# Docker images for dbt are used in a number of important places throughout the dbt ecosystem.  This is how we keep those images up-to-date.
+
+# **when?**
+# This is triggered manually 
+
+# **TODO**
```

Comment:
I'm not trying to be a pest pointing out the TODO and if they are for a follow up PR just close all these out lol

Reply:
They are all for a follow up PR and should be considered nice-to-haves.  I've updated the wordage so that's a bit more clear.